src/libs/NetworkConnection.js

```diff
@@ -78,8 +78,16 @@ function subscribeToNetInfo() {
             // By default, NetInfo uses `/` for `reachabilityUrl`
             // When App is served locally (or from Electron) this address is always reachable - even offline
             // Using the API url ensures reachability is tested over internet
-            reachabilityUrl: `${CONFIG.EXPENSIFY.DEFAULT_API_ROOT}api`,
-            reachabilityTest: response => Promise.resolve(response.status === 200),
+            reachabilityUrl: `${CONFIG.EXPENSIFY.DEFAULT_API_ROOT}api?command=Ping`,
```

Comment:
Let's use Get rvl=nameValuePairs value=private_lastModified

Reply:
This doesn't work because it requires an Auth token + a POST request